nextflow.config

```diff
@@ -23,6 +23,7 @@ params {
     strand = '0'
     mode = 'single'
     tpm = 1
+    fastp_additional_params = ''
```

Comment:
@MarieLataretu this should also become a parameter in the `main.nf`, or? You introduced this to provide the adapter sequence, right?

Reply:
yes, I add `fastp_additional_params = '-a TGGAATTCTCGGGTGCCAAG'` in the data set specific config file (for the sRNA data sets)

Reply:
I'd do the same for additional `HISAT2` parameters!

Reply:
yep, sounds good! and makes more sense then mirroring all the parameters of a tool